### ISSUE TITLE
CLI: how to validate or describe a JSON file?

### ISSUE THREAD

**Issue by aborruso** [frictionlessdata/frictionless-py#827]
# Overview

Hi,
I have this file

```json
[{ "fieldA": "a", "fieldB": 1, "fieldC": "luke" },
{ "fieldA": "b", "fieldB": 7, "fieldC": "sarah" },
{ "fieldA": "a", "fieldB": 18, "fieldC": "ben" }]
```

If I run `frictionless describe input.json`, I have

```
[resource-error] The data resource has an error: cannot extract metadata "input.json" because ""
```

What I must add to my command?

Thank you

---

Please preserve this line to notify @roll (lead of this repository)

**Comment by mikenac** [comment]
I would also like to understand this. Frictionless looks interesting, but is it only for CSV data?

**Comment by aborruso** [comment]
> I would also like to understand this. Frictionless looks interesting, but is it only for CSV data?

No, for a lot of input "things" https://framework.frictionlessdata.io/docs/tutorials/formats/bigquery-tutorial

![image](https://user-images.githubusercontent.com/30607/116253821-1b886a00-a771-11eb-8458-b0b2cf7abc72.png)

**Comment by aborruso** [comment]
In the [FAQ](https://framework.frictionlessdata.io/docs/faq#why-am-i-getting-the-error-cannot-extract-tabular-data-from-json) we have

> Nowadays, Frictionless treats all JSON and YAML files as metadata

But it should be able to [read also json data](https://framework.frictionlessdata.io/docs/tutorials/formats/json-tutorial)

**Comment by roll** [comment]
Hi,

Currently, with JSON/YAML data, it works this way:

```python
extract('data/table.json') # will fail as it detects it to be a resource metadata
extract(path='data/table.json') # will work as we explicitly provide a data path
```

Since `frictionless@4.9`, there is the `--path` argument for the CLI doing to same:

```bash
$ frictionless extract data/table.json # will fail as it detects it to be a resource metadata
$ frictionless extract --path data/table.json # will work as we explicitly provide a data path
```

It was exposed to all of `describe/extract/validate`

**Comment by aborruso** [comment]
> It was exposed to all of `describe/extract/validate`

If I run `frictionless describe --path input.json`, I have

```
Usage: frictionless describe [OPTIONS] [SOURCE]...
Try 'frictionless describe --help' for help.

Error: no such option: --path
```

I'm using `4.8.1`

**Comment by aborruso** [comment]
I have the same for `extract`

**Comment by aborruso** [comment]
Ah, sorry, since the next release. Sorry again

**Comment by aborruso** [comment]
@roll it works, thank you!

**Comment by roll** [comment]
Yea it's just landed on Pypi (`frictionless@4.9`)
